mex/extractors/datenkompass/models/item.py

```diff
@@ -43,19 +43,19 @@ class DatenkompassBibliographicResource(BaseModel):
 
     model_config = {"populate_by_name": True}
 
-    beschreibung: str | None = Field(None, alias="Beschreibung")
+    beschreibung: list[str] | None = Field(None, alias="Beschreibung")
```

Comment:
```suggestion
    beschreibung: list[str]  = Field([], alias="Beschreibung")
```
The list equivalent of None is an empty list, right?

Reply:
Same for the fields below